bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingFactory.java

```diff
@@ -256,5 +273,6 @@ private static Object getDefaultValueAsCorrectType(Type parameterType, String de
             return null;
         }
     }
+    
```

Comment:
This one could be removed, there is already one empty line. I don't think we need two.